keyboards/trkeyboards/trk1/readme.md

```diff
@@ -0,0 +1,30 @@
+# Tvrd Rad Keyboards TRK1
+
+![TRK1](https://www.trkeyboards.com/upload/products/gallery/1663968347286_raaw8387.jpg)
```

Comment:
Please rehost the image on a dedicated image host, such as Imgur (with an account)

No offense, but shops are ephemeral, and regularly vanish.